src/Operation/ChangeStreamCommand.php

```diff
@@ -0,0 +1,195 @@
+ * limitations under the License.
+ */
+
+namespace MongoDB\Operation;
+
+use MongoDB\ChangeStream;
+use MongoDB\Driver\Manager;
+use MongoDB\Driver\ReadConcern;
+use MongoDB\Driver\ReadPreference;
+use MongoDB\Driver\Server;
+use MongoDB\Driver\Exception\RuntimeException as DriverRuntimeException;
+use MongoDB\Exception\InvalidArgumentException;
+use MongoDB\Exception\UnexpectedValueException;
+use MongoDB\Exception\UnsupportedException;
+use ArrayIterator;
+use stdClass;
+use Traversable;
+
+/**
+ * Raw changeStream command.
+ *
+ * @api
+ * @see \MongoDB\Collection::changeStream()
+ * @see http://docs.mongodb.org/manual/reference/command/changeStream/
+ */
+class ChangeStreamCommand implements Executable
+{
+    const FULL_DOCUMENT_DEFAULT = 'default';
+    const FULL_DOCUMENT_UPDATE_LOOKUP = 'updateLookup';
+
+    private $databaseName;
+    private $collectionName;
+    private $pipeline;
+    private $options;
+    private $manager;
+
+    /**
+     * Constructs a changeStream command.
+     *
+     * Supported options:
+     *
+     *  * fullDocument (string): Allowed values: ‘default’, ‘updateLookup’.
+     *    Defaults to ‘default’.  When set to ‘updateLookup’, the change
+     *    notification for partial updates will include both a delta describing
+     *    the changes to the document, as well as a copy of the entire document
+     *    that was changed from some time after the change occurred. For forward
+     *    compatibility, a driver MUST NOT raise an error when a user provides
+     *    an unknown value. The driver relies on the server to validate this
+     *    option.
+     *
+     *  * resumeAfter (document): Specifies the logical starting point for the
+     *    new change stream.
+     *
+     *  * readConcern (MongoDB\Driver\ReadConcern): Read concern. Note that a
+     *    "majority" read concern is not compatible with the $out stage
+     *
+     *    This is not supported for server versions < 3.2 and will result in an
+     *    exception at execution time if used.
+     *
+     *  * readPreference (MongoDB\Driver\ReadPreference): Read preference.
+     *
+     *  * maxAwaitTimeMS (integer): The maximum amount of time for the server to
+     *    wait on new documents to satisfy a change stream query.
+     *
+     *  * batchSize (integer): The number of documents to return per batch.
+     *
+     *    This option is sent only if the caller explicitly provides a value.
+     *    The default is to not send a value.
+     *
+     *  * collation (document): Specifies a collation.
+     *
+     *    This option is sent only if the caller explicitly provides a value.
+     *    The default is to not send a value.
+     *
+     * @param string         $databaseName   Database name
+     * @param string         $collectionName Collection name
+     * @param array          $pipeline       List of pipeline operations
+     * @param array          $options        Command options
```

Comment:
`@param Manager $manager Manager instance from the driver` can be added after `$options`.